api/common/network_test.go

```diff
@@ -430,34 +424,11 @@ func (s *NetworkSuite) TestGetObservedNetworkConfigAddressNotInCIDRFormat(c *gc.
 	s.stubConfigSource.CheckCall(c, 3, "InterfaceAddresses", "eth0")
 }
 
-func (s *NetworkSuite) TestGetObservedNetworkConfigEmptyAddressValue(c *gc.C) {
```

Comment:
This logic is now pushed into the config source from the `GetObservedNetworkConfig` method. A following patch will make some changes there and re-implement this test as appropriate.